flexport/latitude
# Fixed DatePicker styling

This PR fixes DatePicker not getting styled due to the ordering of the loading of css assets. 
This PR also removes the SSR conditional since the comment documenting its purpose is no longer correct and SSR isn't used in the monorepo.

Manual Testing Plan:

Open storybook, 
navigate to the CalendarDateRange story,
ensure the DatePicker is styled.
![image](https://user-images.githubusercontent.com/5897853/67432906-0343f780-f59c-11e9-9a6b-56dddf6ca371.png)
